website/server/models/user/schema.js

```diff
@@ -108,9 +108,9 @@ let schema = new Schema({
 
   balance: {type: Number, default: 0},
   // Not saved on the user right now
-  filters: {type: Schema.Types.Mixed, default: () => {
-    return {};
-  }},
+  // filters: {type: Schema.Types.Mixed, default: () => {
```

Comment:
@crookedneighbor no we don't. I asked for this to be commented out because we always left it in the code in case we decided to start persisting them again... but yeah it can be removed